spring-grpc-core/src/main/java/org/springframework/grpc/client/GrpcChannelFactory.java

```diff
@@ -16,37 +16,32 @@
 
 package org.springframework.grpc.client;
 
-import java.util.List;
-
+import io.grpc.ChannelCredentials;
 import io.grpc.ClientInterceptor;
-import io.grpc.ManagedChannelBuilder;
+import io.grpc.Grpc;
+import io.grpc.ManagedChannel;
 
 /**
- * Factory interface for creating {@link ManagedChannelBuilder} instances for a given
- * authority.
+ * Factory interface for creating {@link ManagedChannel} instances.
  *
  * @author Dave Syer
- * @see ManagedChannelBuilder
+ * @author Chris Bono
  */
 public interface GrpcChannelFactory {
 
 	/**
-	 * Creates a {@link ManagedChannelBuilder} for the given authority.
-	 * @param authority the target authority for the channel
-	 * @return a {@link ManagedChannelBuilder} configured for the given authority
-	 */
-	ManagedChannelBuilder<?> createChannel(String authority);
```

Comment:
I think maybe we should keep this overload. It can have a default implementation.